docs/README.md

```diff
@@ -1,5 +1,43 @@
 # API documents with [API blueprint](https://apiblueprint.org/)
 
+- https://apiblueprint.org/documentation/tutorial.html
+
+Install [aglio](https://github.com/danielgtaylor/aglio):
+
+```
+brew install node
+npm config set prefix '~/.npm-global'
+export PATH=~/.npm-global/bin:$PATH
+npm install -g aglio
```

Comment:
I think global installation is not good for non-nodejs developers.

There are two options.

1. Intall locally `npm install aglio`. and then you can run `./node_modules/.bin/aglio`.
2. Just use `npx aglio` if there is npx command in nodejs. It run `aglio` instantly with installation.

Reply:
`npx aglio` is better!

Reply:
I will change it with `npx aglio`